vue-frontend/src/components/Home.vue

```diff
@@ -5,20 +5,23 @@
       <Header />
       <HomeLandingPage />
       <HomeClientReviewPage />
+      <HomeUserReviewPage />
     </div>
   </div>
 </template>
 
 <script>
 import Header from "./partials/Header.vue";
 import HomeLandingPage from "./home/HomeLandingPage.vue";
+import HomeUserReviewPage from "./home/HomeUserReviewPage.vue";
 import HomeClientReviewPage from "./home/HomeClientReviewPage.vue";
```

Comment:
I think we can refactor component name to `ClientReview`, as it's already on path /home.